javalib/src/main/scala/java/util/StringTokenizer.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package java.util
+
+class StringTokenizer(
+  str: String,
+  private var delim: String,
+  returnDelims: Boolean
+) extends java.util.Enumeration[Object] {
+
+  def this(str: String) = this(str, " \t\n\r\f", false)
+  def this(str: String, delim: String) = this(str, delim, false)
+
+  private var position: Int = 0
+  private val length: Int = str.length
+
+  def countTokens(): Int = {
+    var count: Int = 0
+    var inToken: Boolean = false
+    var i: Int = position
```

Comment:
It's arguable whether to use explicit types for *local* `var`s, but usually we omit them. We sometimes use them when it's not obvious what the type should be, or if the initial value as a more restrictive type (like `None` but we need `Option[Int]`). In this case I think they lead to more clutter than necessary, but I have no strong feeling about it.